sys/coreenv/src/Config.hs

```diff
@@ -33,16 +36,22 @@ data SolverConfig = SolverConfig
 newtype SolverId = SolverId { solverId :: String }
   deriving (Eq, Ord, Show)
 
+newtype ParameterName = ParamName { getParamName :: String }
```

Comment:
We should be consistent with the naming: either `ParameterName` or `ParamName` (to me the shorter the better :)).

Also we could get rid of the `get` part, since we don't use this prefix in all the other accesor functions (`solverId`, `smtLog`, etc).

Reply:
Fixed